src/avro2json.c

```diff
@@ -1036,22 +1072,49 @@ static const char *parse_args(int argc, char **argv, config_t *conf) {
     } else if (!strcmp(argv[arg_idx], "--csv")) {
       conf->output_csv = 1;
     } else if (!strcmp(argv[arg_idx], "--columns") && arg_idx < argc - 1) {
```

Comment:
I'd suggest retaining the old logic and using another command line argument for the new functionality (`--column-names`?)

Reply:
It is not going to be used, but it's a maintenance hassle. We anyway have every logic in 2 flows (json and csv), so extending it to 4 flows... I don't see much point.

Reply:
Also, if we extend the cmdline to pass mapping transformations (and we likely will), it will be easier to pass tuples together with the names, something like --columns '[["XTime","DateTimeFromUnixNanoseconds"], "RowGuid", "XFloat"]'